### PR TITLE
Issue #1442: Error Before Confirm Action

### PR DESCRIPTION
**Pull request by sgershuny**
Issue #1442: - Error shows before action confirmed on withdrawal and send statecoin

### CODE REVIEW

File: src/features/WalletDataSlice.js [hunk 1]
```diff
@@ -575,6 +575,56 @@ export const setIntervalIfOnline = (func, online, delay) => {
...
+
+  if(callIsBatchMixedPrivacy(selectedCoins)) {
+    dispatch(setNotification({msg:"Warning: Withdrawal transaction contains both private and un-swapped inputs."}))
+  }
+}
+
+export const checkSend = (dispatch, inputAddr ) => {
+  // Pre action confirmation checks for send statecoin - return true to prevent action
+
+  var input_pubkey = "";
```
Comment (lawlawlaw) [inline src/features/WalletDataSlice.js:607]: The address validity should be checked before the confirmation dialog is shown. Therefore the address should be check in checkWithdrawal().

**Review by lawlawlaw — CHANGES_REQUESTED** [review]
See comment. Please add tests,

File: src/features/WalletDataSlice.js (hunk 1 above)
Comment (sgershuny) [reply src/features/WalletDataSlice.js:607]: I've had to revert the change I made as using the "encodeSCEAddress" function to check a btc address always returns an error

Comment (lawlawlaw) [reply src/features/WalletDataSlice.js:607]: It still doesn't check for a problem with the address before displaying the confirmation dialog.

**Pull request merged**